jenkinsfiles/canary

```diff
@@ -0,0 +1,75 @@
+@Library('pipeline-library') _
+
+pipeline {
+    agent {
+        label 'ec2-jdk8'
+    }
+
+    triggers {
+        cron('H 20 * * 1-5')
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+        timeout(time: 20, activity: true)
+    }
+
+    environment {
+        MAVEN_OPTS = '-Xms1024m -Xmx4096m'
+    }
+
+    stages {
+        stage ('Build') {
+            steps {
+                echo 'Building DHIS2 ...'
+                script {
+                    withMaven(options: [artifactsPublisher(disabled: true)]) {
+                        sh 'mvn -X -T 4 clean install -f dhis-2/pom-full.xml -Pjdk8 --update-snapshots'
+                    }
+                }
+            }
+        }
+
+        stage ('Sync WAR') {
+            steps {
+                echo 'Syncing WAR ...'
+                sh 'curl "https://raw.githubusercontent.com/radnov/dhis2-server-setup/master/ci/scripts/sync-war-s3.sh" -O'
```

Comment:
should we use dhis2/dhis2-server-setup here?

Reply:
I didn't want to create a new script with a similar name on the `dhis2/dhis2-server-setup` upstream repo, so I've used my fork to avoid breaking any current usage of the script (on the old Jenkins instance). It's either that or using a different branch on the upstream repo.

Reply:
I think I would just use a new name. `copy-war-s3.sh` ?
Then we can just remove the old one when we want.